changelog/next/features/3064--operator-aliases.md

```diff
@@ -0,0 +1,7 @@
+User-defined operator aliases make pipelines easier to use by enabling users to
+encapsulates pipelines into a new operator. To define a user-defined operator
```

Comment:
```suggestion
encapsulate pipelines into a new operator. To define a user-defined operator
```

Reply:
This, plus `a pipeline` instead of `pipelines`.

Reply:
Nice catch, fixed in a15465b4a